test.js

```diff
@@ -132,6 +135,11 @@ test('test WriteStream with async writes', function (t, ctx, done) {
     }
   }
 
+  // Each ws.write()/process.nextTick() pair above cause a batch
+  var next = after(sourceData.length + 1, ctx.verify.bind(ctx, ws, done))
```

Comment:
This test was a bit tricky at first, but it turns out that an array of 10 elements will yield 10 batch operations, due to `process.nextTick()`. Hence we wait for 11 functions to be called, 10 + 1 for the `'close'` event in the stream.